configure

```diff
@@ -5999,17 +5999,31 @@ append_var SONIVOX_LIBS "-lsonivox"
 if test "$_sonivox" = auto; then
 	_sonivox=no
 	cat > $TMPC << EOF
+#include <stdio.h>
```

Comment:
No need of stdio I believe?

Reply:
just to be explicit for the NULL symbol, and I see that we usually include it in most tests, even the ones not using stdio.

but I can drop it if you like, `eas.h` probably has what's required for NULL anyway

Reply:
Ah. If it's for NULL, please use stdlib.h.
That's lighter.
```suggestion
#include <stdlib.h>
```

Reply:
Sure! I was also thinking about stddef.h.

About to amend this